Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -1660,12 +1689,14 @@ def get_items_from_folder(folder_path, limit=100, target_mailbox=None, is_public
         item_attachment = parse_item_as_dict(item, account.primary_smtp_address, camel_case=True,
                                              compact_fields=True)
         for attachment in item.attachments:
-            if get_internal_item and isinstance(attachment, ItemAttachment) and isinstance(attachment.item,
-                                                                                           Message):
-                # if found item attachment - switch item to the attchment
-                item_attachment = parse_item_as_dict(attachment.item, account.primary_smtp_address, camel_case=True,
-                                                     compact_fields=True)
-                break
+            if attachment is not None:
+                attachment.parent_item = item
```

Comment:
what do we need this for?

Reply:
It seems this is just good practice so I left it in: https://github.com/ecederstrand/exchangelib/issues/210
Can take it out if needed